modules/final_radiological_review/php/NDB_Form_final_radiological_review.class.inc

```diff
@@ -171,6 +172,14 @@ class NDB_Form_final_radiological_review extends NDB_Form
                 );
             }
 
+            $radiology_review_table_exists = $DB->pselect("SHOW TABLES LIKE 'radiology_review'", array());
+            if (count($radiology_review_table_exists) === 0) {
```

Comment:
@Jkat 

you could use this instead:
$rad_rev_table_exists=$DB->tableExists('radiology_review')
if ($rad_rev_table_exists) {
...
}